crates/database/src/surplus_capturing_jit_order_owners.rs

```diff
@@ -0,0 +1,56 @@
+use {
+    crate::{auction::AuctionId, Address},
+    sqlx::PgConnection,
+};
+
+pub async fn insert(
```

Comment:
shouldn't this be in the `persistence`?

Reply:
this is a standard write function to db, it will be called from `persistence`